Network/Wai/Middleware/Static.hs

```diff
@@ -56,6 +59,26 @@ import qualified System.FilePath as FP
 newtype Policy = Policy { tryPolicy :: String -> Maybe String -- ^ Run a policy
                         }
 
+-- | Options for 'staticWithOptions' 'Middleware'.
+data Options = Options { cacheContainer :: CacheContainer -- ^ Cache container to use
+                       , mimeTypes :: FilePath -> MimeType -- ^ Compute MimeType from file name
+                       }
+
+-- | Default options.
+--
+-- 'cacheContainer' = 'CacheContainerEmpty' -- no caching
+-- 'mimeTypes' = 'getMimeTypes' -- use 'defaultMimeLookup' from 'Network.Mime'
```

Comment:
Having these two lines appear consecutively with no whitespace between them will cause them to be rendered poorly in Haddocks. I would suggest doing one of the following instead:

* Rendering it with code, like [`Data.Aeson.Options` does](https://hackage.haskell.org/package/aeson-1.4.5.0/docs/Data-Aeson.html#v:defaultOptions):

  ```hs
  -- @
  -- 'Options'
  -- { 'cacheContainer' = 'CacheContainerEmpty' -- no caching
  -- , 'mimeTypes'      = 'getMimeTypes'        -- use defaultMimeLookup from Network.Mime
  -- }
  -- @
  ```
* Putting each selector in its own entry in a bulleted list:

  ```hs
  -- * @'cacheContainer' = 'CacheContainerEmpty' -- no caching@
  -- * @'mimeTypes'      = 'getMimeTypes'        -- use defaultMimeLookup from Network.Mime@
  ```